impress-org/givewp
# Refactor: change the global settings link for the terms and conditions block and add it for name title prefixes on the name block

## Description



The Title Prefixes options for the name block and the Label and Text options for the Terms and Conditions block have global settings that should be considered.

This PR implements a global settings link for the Name block and adds a few improvements for the Terms and Conditions block that already had it implemented.

This PR also implements a few minor changes in the settings page to display these content options all the time, this way we can ensure that they will not be hidden when the user accesses the settings page through the links from the visual form builder.

## Affects



The name block and the terms and conditions block.

## Visuals



**Name block:**

![image](https://github.com/impress-org/givewp/assets/16308864/0ad96ef4-18c8-4819-950d-022024e95300)

![image](https://github.com/impress-org/givewp/assets/16308864/3f69fdd1-d9e2-4811-93da-50a1091b2937)

![image](https://github.com/impress-org/givewp/assets/16308864/32a73dcc-198f-459f-babd-a34626477252)

**Terms and Conditions block:** 

![image](https://github.com/impress-org/givewp/assets/16308864/2d9cdf5d-d046-4537-be42-6d626f70fc50)

![image](https://github.com/impress-org/givewp/assets/16308864/ddc246cf-2ed3-476e-8346-b9cab58a0c1f)


## Testing Instructions



1. Created a form with the visual form builder;
2. Add the Terms and Conditions Block and select the global options, the link below the select input now should inform which options can be edited in the global settings, and when acces it, the options should be visible;
3. In the name block enable the Show Name Title Prefix and notice that now you'll see a select input with the `"Global"` and `"Customize"` options, play around with these ones.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205920479946985
  - https://app.asana.com/0/0/1205076251265794
